src/rpc/handlers/AccountTx.cpp

```diff
@@ -105,15 +105,20 @@ AccountTxHandler::process(AccountTxHandler::Input input, Context const& ctx) con
     if (input.ledgerHash || input.ledgerIndex || input.usingValidatedLedger)
     {
         if (ctx.apiVersion > 1u && (input.ledgerIndexMax || input.ledgerIndexMin))
+        {
             return Error{Status{RippledError::rpcINVALID_PARAMS, "containsLedgerSpecifierAndRange"}};
+        }
+        else if (!input.ledgerIndexMax && !input.ledgerIndexMin)
+        {  // mimic rippled, when both range and index specified, respect the range.
```

Comment:
Hm, i'd hope `clang-format` would fix this automatically - comments should be on a separate line from `{` and aligned with the start of `auto const lgrInfoOrStatus` line.